components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/APIMappingUtil.java

```diff
@@ -825,4 +845,40 @@ public static AdvertiseInfoDTO extractAdvertiseInfo(API api) {
         return advertiseInfoDTO;
     }
 
+    /**
+     * Checks whether tenant is allowed to subscribe
+     *
+     * @param apiTenant                  Tenant of the API creator
+     * @param subscriptionAvailability   Subscription availability
+     * @param subscriptionAllowedTenants Subscription allowed tenants
+     * @return subscriptionAllowed
+     */
+    private static boolean isSubscriptionAvailable(String apiTenant, String subscriptionAvailability,
+                                                  String subscriptionAllowedTenants) {
+
+        String userDomain = RestApiUtil.getLoggedInUserTenantDomain();
```

Comment:
please rename this to tenantDOmain, otherwise it sounds like userstore domain.

Reply:
Fixed